pkg/operation/botanist/controlplane/etcd/bootstrap.go

```diff
@@ -1,12 +1,310 @@
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      druidServiceAccountName,
+				Namespace: namespace,
+				Labels:    map[string]string{v1beta1constants.GardenRole: Druid},
+			},
+		}
+		serviceAccountYAML, _ = runtime.Encode(codec, serviceAccount)
+
+		clusterRole = &rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   druidRBACName,
+				Labels: map[string]string{v1beta1constants.GardenRole: Druid},
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups: []string{corev1.GroupName},
+					Resources: []string{"pods"},
+					Verbs:     []string{"list", "watch", "delete"},
+				},
+				{
+					APIGroups: []string{corev1.GroupName},
+					Resources: []string{"secrets", "endpoints"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch"},
+				},
+				{
+					APIGroups: []string{corev1.GroupName},
+					Resources: []string{"events"},
+					Verbs:     []string{"create", "get", "list", "watch", "patch", "update"},
+				},
+				{
+					APIGroups: []string{corev1.GroupName, appsv1.GroupName},
+					Resources: []string{"services", "configmaps", "statefulsets"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
+				},
+				{
+					APIGroups: []string{druidv1alpha1.GroupVersion.Group},
+					Resources: []string{"etcds"},
+					Verbs:     []string{"get", "list", "watch", "update", "patch"},
+				},
+				{
+					APIGroups: []string{druidv1alpha1.GroupVersion.Group},
+					Resources: []string{"etcds/status", "etcds/finalizers"},
+					Verbs:     []string{"get", "update", "patch", "create"},
+				},
+			},
+		}
+		clusterRoleYAML, _ = runtime.Encode(codec, clusterRole)
+
+		clusterRoleBinding = &rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   druidRBACName,
+				Labels: map[string]string{v1beta1constants.GardenRole: Druid},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "ClusterRole",
+				Name:     druidRBACName,
+			},
+			Subjects: []rbacv1.Subject{
+				{
+					Kind:      rbacv1.ServiceAccountKind,
+					Name:      druidServiceAccountName,
+					Namespace: namespace,
+				},
+			},
+		}
+		clusterRoleBindingYAML, _ = runtime.Encode(codec, clusterRoleBinding)
+
+		configMapImageVectorOverwrite = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      druidConfigMapImageVectorOverwriteName,
+				Namespace: namespace,
+				Labels:    map[string]string{v1beta1constants.GardenRole: Druid},
+			},
+		}
+		configMapImageVectorOverwriteYAML []byte
+
+		vpaUpdateMode = autoscalingv1beta2.UpdateModeAuto
+		vpa           = &autoscalingv1beta2.VerticalPodAutoscaler{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      druidVPAName,
+				Namespace: namespace,
+				Labels:    map[string]string{v1beta1constants.GardenRole: Druid},
+			},
+			Spec: autoscalingv1beta2.VerticalPodAutoscalerSpec{
+				TargetRef: &autoscalingv1.CrossVersionObjectReference{
+					APIVersion: appsv1.SchemeGroupVersion.String(),
```

Comment:
Doesn't the deployment apiVersion have to be pluggable anymore (as in `{{ include "deploymentversion" . }}`)?

Reply:
At the moment: No, all supported Kubernetes versions are compatible with `apps/v1`.